src/components/GWL.vue

```diff
@@ -617,38 +729,47 @@ export default {
 #bkgrd-map-grp {
   filter: drop-shadow(0.2rem 0.2rem 0.5rem rgba(38, 49, 43, 0.15));
   stroke-width: 0.2;
-  color: "white"
+  color: white;
+  fill: white;
 }
 // annotated timeline
 .liney {
   stroke-width: 2px;
-  color: rgb(143, 206, 131);
 }
 
 // desktop
```

Comment:
I designed the layout mobile first, which I think helped. Ideally everything is in view at once regardless of screen size. I've adjusted the height parameter for the desktop version so it fills the space in the wide version.

Reply:
Ugh I wish editing these grids for different windows wasn't so fiddly! The chart is now drawing on my laptop screen, so that's great. Though now on my full monitor view, you have to scroll down to see the chart which isn't ideal: 
![image](https://user-images.githubusercontent.com/54007288/139701042-7035d7f3-c675-46ca-84f6-2b174baf75f0.png)

Reply:
Here's how it looks with recent changes on laptop:
![image](https://user-images.githubusercontent.com/17803537/139755304-bb4b1690-e4d4-4e68-b82d-735c878d08df.png)
I adjusted for relative font scaling and set a max-height on desktop using vh